geoutils/georaster/raster.py

```diff
@@ -736,10 +719,7 @@ def __pow__(self: RasterType, power: int | float) -> RasterType:
         # Calculate the product of arrays and save to new Raster
         out_data = self.data**power
         ndv = self.nodata
```

Comment:
You can skip setting `ndv` and use directly `self.nodata` in the following line.